website/docusaurus.config.js

```diff
@@ -1,215 +1,41 @@
-          editUrl: 'https://github.com/visgl/math.gl/tree/master/website',
-        },
-        theme: {
-          customCss: [
-            resolve('./src/styles.css'),
-            // resolve('./node_modules/maplibre-gl/dist/maplibre-gl.css')
-          ],
-        },
-      }),
-    ]
-  ],
-
-  plugins: [
-    [
-      './ocular-docusaurus-plugin',
-      {
-        debug: true,
-        resolve: {
-          modules: [resolve('node_modules'), resolve('../node_modules')],
-          alias: {
-            'website-examples': resolve('../examples'),
-
-            '@math.gl/types': resolve('../modules/types/src'),
-            '@math.gl/core': resolve('../modules/core/src'),
-            '@math.gl/culling': resolve('../modules/culling/src'),
-            '@math.gl/geospatial': resolve('../modules/geospatial/src'),
-            '@math.gl/geoid': resolve('../modules/geoid/src'),
-            '@math.gl/polygon': resolve('../modules/polygon/src'),
-            '@math.gl/proj4': resolve('../modules/proj4/src'),
-            '@math.gl/web-mercator': resolve('../modules/web-mercator/src'),
-            '@math.gl/sun': resolve('../modules/sun/src'),
-            '@math.gl/ddgs-geohash': resolve('../modules/ddgs-geohash/src'),
-            '@math.gl/dggs-quadkey': resolve('../modules/dggs-quadkey/src'),
-            '@math.gl/dggs-s2': resolve('../modules/dggs-s2/src'),
-
-            '@probe.gl/bench': resolve('../node_modules/@probe.gl/bench')
-          }
-        },
-        plugins: [
-          // new webpack.EnvironmentPlugin(['MapboxAccessToken', 'GoogleMapsAPIKey', 'GoogleMapsMapId']),
-          // These modules break server side bundling
-          new webpack.IgnorePlugin({
-            resourceRegExp: /asciify-image/
-          })
-        ],
-        module: {
-          rules: [
-            // https://github.com/Esri/calcite-components/issues/2865
-            {
-              test: /\.m?js/,
-              resolve: {
-                fullySpecified: false
-              }
-            }
-          ]
-        }
-      }
-    ],
-    [
-      '@docusaurus/plugin-content-docs',
-      {
-        id: 'examples',
-        path: './src/examples',
-        routeBasePath: 'examples',
-        sidebarPath: resolve('./src/examples-sidebar.js'),
-        breadcrumbs: false,
-        docItemComponent: resolve('./src/components/example/doc-item-component.jsx')
-      },
-    ],
-    [
-      require.resolve('@cmfcmf/docusaurus-search-local'),
-      {
-        // Options here
+  siteUrl: 'https://visgl.github.io/math.gl',
+  repoUrl: 'https://github.com/visgl/math.gl',
+
+  docsTableOfContents: require('../docs/table-of-contents.json'),
+
+  // examplesDir: './src/examples',
+  // exampleTableOfContents: require('./src/examples/table-of-contents.json'),
+
+  search: 'local',
+
+  webpackConfig: {
+    resolve: {
+      alias: {
+        'website-examples': resolve('../examples'),
+
+        '@math.gl/types': resolve('../modules/types/src'),
```

Comment:
We could probably automate alias generation for `modules`? Maybe as an option to `getDocusaurusConfig()` if we are concerned about difference between monorepos and single module repos? 
Would reduce the size of config here and also automatically catch any new modules added.